docs/platform/howto/byoc/assign-project-custom-cloud.md

```diff
@@ -1,24 +1,19 @@
 ---
 title: Enable your AWS custom cloud in Aiven organizations, units, or projects
+sidebar_label: Attach projects
 ---
 
 To use a [custom cloud](/docs/platform/concepts/byoc) in your Aiven organizations, units, or projects, configure its availability.
 
-:::important
-Custom cloud configuration in Aiven is an
-[early availability feature](/docs/platform/concepts/beta_services). You cover the costs associated with building and
-maintaining your custom cloud payments for your integrated AWS
-infrastructure and Aiven services within the custom cloud.
-:::
-
 ## About making custom clouds available from your projects
 
 With the BYOC feature enabled, you can
-[create custom clouds](/docs/platform/howto/byoc/create-custom-cloud) in your Aiven organization. As a part of the
-[initial custom cloud's setup in Aiven Console](/docs/platform/howto/byoc/create-custom-cloud), you select in what projects you'll be able to use your new
-custom cloud to create services and make your
-cloud available for all the projects in your organization, selected
-organizational units, or specific projects only.
+[create custom clouds](/docs/platform/howto/byoc/create-custom-cloud) in your Aiven
+organization. As a part of the
+[initial custom cloud's setup in Aiven Console](/docs/platform/howto/byoc/create-custom-cloud),
+you select in what projects you'll be able to use your new custom cloud to host Aiven
+services. You can decide to make your cloud available for all the projects in your
+organization, selected organizational units, or specific projects only.
```

Comment:
That's a duplication of what I commented in another file in this PR, this pattern should be proscribed 😬 
I would argue this is valuable here, but not in the task below, as I imagine it's in the UI?